server/runtime_go_nakama.go

```diff
@@ -2202,7 +2201,7 @@ func (n *RuntimeGoNakamaModule) LeaderboardCreate(ctx context.Context, id string
 		return errors.New("expects a leaderboard ID string")
 	}
 
-	sort := LeaderboardSortOrderDescending
+	var sort int
```

Comment:
I'd say that in this case the problem is that the initial assignment is not very useful since it gets completely overridden in the following switch statement. To your point, the same clarity can be achieved by introducing an enum type alias so that's it's clear it's not a plain integer value here but an enum. Would this solve your concern, Simon?

Reply:
I'm also in favour of the extra readability. Even if the value is overridden shortly afterwards it's an indicator that this should be the "base" case. If there's another way to achieve that I'm open to that too.

Reply:
So I was thinking of something like this:
```go
type LeaderboardSort = int

const (
	LeaderboardSortOrderAscending LeaderboardSort = iota
	LeaderboardSortOrderDescending
)

...

var sort LeaderboardSort
	switch sortOrder {
...
```

this way it's clear that we're dealing with enum and not a generic int, plus, given that the switch below is exhaustive, we don't really need a base case. In short, imho there's no loss of readability with this approach, but, if the consensus is to keep it as it was, I'll be happy to throw some annotations instead :+1:

Reply:
Okay, I reverted the "base-case" implementation and added annotations to ignore these.